tests/vmi_networking_test.go

```diff
@@ -659,11 +659,6 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 		}
 
 		Context("[Conformance][test_id:1780][label:masquerade_binding_connectivity]should allow regular network connection", func() {
-			const (
```

Comment:
How this commit is related to this giant (already very hard to review) PR?

Reply:
It might not be, but this was hurting my eyes, and I had to work in this test.

If you want, I can take it out, and push another PR. Your call.

Reply:
s/test/file/

Reply:
Yes, ease move it to a separate PR.